new-pipeline/src/dashboards.js

```diff
@@ -1081,6 +1081,23 @@ function getDescriptionLink(metric, channel, description) {
       class: "btn btn-outline-primary fa fa-info-circle",
     }).text(" More details"));
   }
-
   return link;
 }
+
+function getUseCounterLink(metric, channel, description) {
+  if (metric.startsWith("USE_COUNTER") !== true) {
+    return null;
+  }
+  else {
```

Comment:
Code style nit: We already `return` above, using an "early return". So we can avoid the indentation here and put the lines below outside of the `else` block, making the function a bit more easy to read.